### DISCUSSION TITLE
Anyone Try Changing Params?

### DISCUSSION THREAD

**Discussion by rpassaro** [kyleskom/NBA-Machine-Learning-Sports-Betting#395] [Ideas]
Quick question/idea, has anyone tried changing the params set in ~/Train-Models/XGBoost_Model_UO.py such as depth, eta, epochs? I'm intrigued to try but I'm not sure if someone has already optimized them (Maybe kyleskom)? Any thoughts or opinions would be appreciated!

**Comment by nealmick** [comment]
I have found model architecture has little impact compared to the data features your training on.  The model is essentially only able to perform to the degree that the data actually captures or represents your target.   I created a dashboard to train models with a bunch of different tensorflow settings. its closed source now but i'll include screenshot of below. 

![Screen Shot 2024-02-14 at 4 51 47 PM](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/33704976/38eda487-9224-4950-a7b3-1819e552b044)

**Reply by rpassaro** [reply]
Cool, thanks a lot
